src/RosInterface.cpp

```diff
@@ -24,6 +24,7 @@ namespace phoxi_camera {
         triggerImageService =nh.advertiseService("trigger_image", &RosInterface::triggerImage, this);
         getFrameService = nh.advertiseService("get_frame", &RosInterface::getFrame, this);
         saveFrameService = nh.advertiseService("save_frame", &RosInterface::saveFrame, this);
+        getScanVolumeService = nh.advertiseService("get_scan_volume", &RosInterface::getScanningVolume, this);
```

Comment:
please rename `get_scan_volume` service to `get_scanning_volume`, thanks

Reply:
Fixed now